lib/include/private/aws_iot_defender_internal.h

```diff
@@ -321,23 +321,28 @@ AwsIotDefenderError_t AwsIotDefenderInternal_BuildTopicsNames( void );
  */
 void AwsIotDefenderInternal_DeleteTopicsNames( void );
 
-/**
- * Connect to AWS with MQTT.
- */
-IotMqttError_t AwsIotDefenderInternal_MqttConnect( void );
-
 /**
  * Subscribe accept/reject defender topics.
  */
-IotMqttError_t AwsIotDefenderInternal_MqttSubscribe( IotMqttCallbackInfo_t acceptCallback,
-                                                     IotMqttCallbackInfo_t rejectCallback );
+
+IotMqttError_t AwsIotDefenderInternal_MqttSubscribe( void );
 
 /**
  * Publish metrics data to defender topic.
  */
 IotMqttError_t AwsIotDefenderInternal_MqttPublish( uint8_t * pData,
                                                    size_t dataLength );
 
+/**
+ * Set shared MQTT connection
+ */
+void AwsIotDefenderInternal_SetMqttConnection( IotMqttConnection_t _mqttConnection );
```

Comment:
Is this function valid anymore (now that the MQTT connection handle is _externed_)?